spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -57,10 +57,17 @@ public LocalModuleDeployer(ModuleLauncher launcher) {
 	@Override
 	public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		String module = request.getCoordinates().toString();
+		if (request.getCount() != 1) {
+			logger.warn(String.format("%s only supports a single instance per module; ignoring count=%d for %s",
```

Comment:
Good to know.
Note that ports are allocated **for (http) `server.port`** only. An FTP listening module wouldn't get that special treatment. Same issue with file paths, etc